modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/background/task/CTPublishBackgroundTaskExecutor.java

```diff
@@ -283,13 +288,40 @@ public String handleException(
 					).put(
 						"ctCollectionName",
 						HtmlUtil.escape(fromCTCollection.getName())
+					).put(
+						"isAdmin", false
 					).put(
 						"notificationType",
 						UserNotificationDefinition.
 							NOTIFICATION_TYPE_REVIEW_ENTRY
 					).put(
 						"showConflicts", showConflicts
 					));
+
+				if (!showConflicts) {
+					_userNotificationEventLocalService.
+						sendUserNotificationEvents(
+							_getAdminUserId(fromCTCollection.getCompanyId()),
+							CTPortletKeys.PUBLICATIONS,
+							UserNotificationDeliveryConstants.TYPE_WEBSITE,
+							false,
+							JSONUtil.put(
+								"backgroundTaskId",
+								backgroundTask.getBackgroundTaskId()
+							).put(
+								"ctCollectionId", fromCTCollectionId
+							).put(
+								"ctCollectionName", fromCTCollection.getName()
+							).put(
+								"isAdmin", true
+							).put(
+								"notificationType",
+								UserNotificationDefinition.
+									NOTIFICATION_TYPE_REVIEW_ENTRY
+							).put(
+								"showConflicts", showConflicts
+							));
+				}
```

Comment:
This logic is actually being added with Brooke's change in https://github.com/brianchandotcom/liferay-portal/pull/147360/commits/a31269bb97268b9e6f7e075f26b50e691f6223a4